docs/1.1-DRAFT/workflows.md

```diff
@@ -169,26 +175,70 @@ A workflow diagram may still be provided even if there is no programmatic `Softw
 
 ## Complying with BioSchemas Computational Workflow profile
 
-To comply with the BioSchemas [ComputationalWorkflow profile],
-where possible, data entities representing _workflows_ SHOULD describe these properties and their related contextual entities:
+Data entities representing _workflows_ (`@type: ComputationalWorkflow`)
+SHOULD comply with the
+BioSchemas [ComputationalWorkflow profile],
+where possible. When complying with this profile, the workflow data entities
+MUST describe these properties and their related contextual entities:
 
 * [name] giving a short descriptive human-readable name for the workflow
 * [programmingLanguage] identifying the workflow system, typed as `ProgrammingLanguage`
 * [creator] identifying the [Person](s) and/or [Organization](s) that assembled the workflow. (This may differ from the [author] of the conceptual workflow, e.g. if the workflow was converted from one language to another).
 * [dateCreated] the date the workflow was first created, e.g. `2020-05-23`
 * [license] identifying a [CreativeWork] that details the license for distributing or editing the workflow
 * [input] and [output] identifying contextual entities for the [FormalParameter] describing input and output parameters/variables that may be varied on different workflow executions
-* [sdPublisher] to identify the [Person] or [Organization] who has made the JSON-LD description of the workflow
-* [url] to identify a public webpage or source code repository for the workflow
+* [sdPublisher] to identify the [Person] or [Organization] who has made this JSON-LD description of the workflow
+* [url] to identify a public page or source code repository for the workflow
```

Comment:
It's confusing throughout, because you also have `@id`!  

I think the nf-core/chipseq example would separate it good:

```json
{
  "@id": "https://raw.githubusercontent.com/nf-core/chipseq/1.2.1/main.nf",
  "@type": ["CreativeWork", "SoftwareSourceCode", "ComputationalWorkflow"],
  "name": "ChIP-seq peak-calling, QC and differential analysis pipeline",
  "alternateName": "nf-core/chipseq",
  "url": "https://nf-co.re/chipseq/1.2.1",
  "version": "1.2.1",
  "codeRepository": {"@id": "https://github.com/nf-core/chipseq"},
  "identifier": "https://doi.org/10.5281/zenodo.3966161",
  "citation": {"@id": "https://doi.org/10.1038/s41587-020-0439-x"}
}
```

For https://schema.org/identifier and https://schema.org/identifier throughout the RO-Crate spec I think we have made them all strings rather than `@id`

Reply:
The citation can be text. In order to make it simpler, why not add the citation as "Ewels, P.A., Peltzer, A., Fillinger, S. et al. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol 38, 276–278 (2020). https://doi.org/10.1038/s41587-020-0439-x"